### PR TITLE
Added privacy policy

### PR DESCRIPTION
**Pull request by vladya32**

### CODE REVIEW

**Comment by katemalash** [comment]
Assign the reviewers when you are making the pool request please

**Comment by YuliaFursovych** [comment]
Could you please merge last 'dev' to this pull-request, because we have removed bootstrap (and related components) and new workflow with scss files.

File: client/src/styles/index.scss [hunk 1]
```diff
@@ -89,12 +89,12 @@ code {
 @import '../components/EventPage/EventPage.scss';
 @import '../components/UI/Spinner/Spinner.scss';
 @import '../components/Rooms/Rooms.scss';
-@import '../components/AddRoom/AddRoom.scss';
```
Comment (YuliaFursovych) [inline client/src/styles/index.scss]: Why did you delete this line?

**Review by YuliaFursovych — APPROVED** [review]

**Review by yodeco — APPROVED** [review]
